download_from_csv.py

```diff
@@ -98,16 +101,35 @@ def download_finish(d):
         'logger': MyLogger(),
         'outtmpl': folder + '/' + song['name'] + ' - ' + song['artist'] + '.%(ext)s'
     }
-    url = ' '.join([song['name'], song['artist'], 'audio', 'youtube'])
+    if args.query == None:
+        url = ' '.join([song['name'], song['artist'], 'audio', 'youtube'])
+    else:
+        url = ' '.join([song['name'], song['artist']]) + ' '
+        url += args.query
     url = 'gvsearch1:' + url
     print '[\033[91mFetching\033[00m] %s' % probable_filename
     with youtube_dl.YoutubeDL(opts) as ydl:
         ydl.download([url])
+
+    image = None
+    if args.art == None:
+        image = album_art.fetch(song, backend='google')
+        if image != None:
```

Comment:
`image` is same as `image != None`

Checkout https://docs.python.org/2.4/lib/truth.html